R/coord-transform.r

```diff
@@ -78,31 +74,25 @@
 #' plot + coord_trans(x = "log10")
 #' plot + coord_trans(x = "sqrt")
 #' }
-coord_trans <- function(x = "identity", y = "identity", limx = NULL, limy = NULL, clip = "on",
-  xtrans, ytrans)
-{
-  if (!missing(xtrans)) {
-    gg_dep("1.0.1", "`xtrans` arguments is deprecated; please use `x` instead.")
-    x <- xtrans
+coord_trans <- function(x = "identity", y = "identity", xlim = NULL, ylim = NULL,
```

Comment:
Can you please follow the advice in https://github.com/tidyverse/principles/issues/67 for deprecating an argument?

You also need to mention the removal of `xtrans` and `ytrans` in the news